CONTRIBUTING.md

```diff
@@ -17,12 +17,36 @@ want a `.war` as quick as possible, you can use this instead:
 $ mvn -Dmaven.test.skip=true clean package 
 ```
 
+
 # Suggesting modifications
 
 Airsonic's source code is hosted on [github](https://github.com/airsonic/airsonic/),
 who provides a [lot of documentation](https://help.github.com/en) on how
 to contribute to projects hosted there.
 
+Keep in mind that we are a small project, and that we value stability over new
+features.
+
+
+# Getting your pull-requests reviewed an merged
+
+Once you issued a pull-request, it will be reviewed by other contributors, and
+modifications/enhancement could be suggested. Once all the concerns are
+addressed, it might get merged, which usually requires two [core
+contributors](https://github.com/airsonic/airsonic/graphs/contributors): one to
+explicitly approve the pull-request, and a second to do the actual merge.
+Note that incremental/small changes are likely going to get reviewed, accepted
+and merged quicker than huge ones.
+
+If you keep sending great code, you might be invited to become a *core
+contributor*.
+
+Releases aren't happening at fixed dates: they happen when enough bugs have
+been fixed, or new features are stable enough. We might delay the inclusion of 
```

Comment:
"or when changes and new features have had enough testing to be considered stable."